addons/mail/static/src/models/popover_view.js

```diff
@@ -249,6 +268,16 @@ registerModel({
             identifying: true,
             inverse: 'reactionPopoverView',
         }),
+        messageNotificationPopoverContentView: one('MessageNotificationPopoverContentView', {
+            compute: '_computeMessageNotificationPopoverContentView',
+            inverse: 'popoverViewOwner',
+            isCausal: true,
+            readonly: true,
```

Comment:
This need rebase ? I think this readonly is not useful anymore in master

Reply:
done